src/lib/ngsi/ContextElementResponse.cpp

```diff
@@ -110,3 +111,43 @@ void ContextElementResponse::present(const std::string& indent, int ix)
   contextElement.present(indent, ix);
   statusCode.present(indent);
 }
+
+
+
+/* ****************************************************************************
+*
+* ContextElementResponse::fill - 
+*/
+void ContextElementResponse::fill(QueryContextResponse* qcrP)
+{
+  if (qcrP == NULL)
+  {
+    statusCode.fill(SccContextElementNotFound);
+    return;
+  }
+
+  if (qcrP->contextElementResponseVector.size() == 0)
+  {
+    statusCode.fill(&qcrP->errorCode);
+    return;
+  }
+
+  //
+  // FIXME P7: If more than one context element is found, we simply select the first one.
```

Comment:
OK, I'm not sure whether you want me to modify anything here ...
Include the issue numbers in the FIXME text?